### ISSUE TITLE
Manual Syncing between Android and desktop app?

### ISSUE THREAD

**Issue by nomatica** [clipto-pro/Desktop#64]
Reading the webpage, it looks like it is possible to manually sync between desktop and android app without creating login. How is this done?

**Comment by atrashler** [comment]
> Reading the webpage, it looks like it is possible to manually sync between desktop and android app without creating login. How is this done?

Where have you read it?:) The app uses Firebase Cloud to store and sync data across all platforms.

**Comment by nomatica** [comment]
Apologies it did not say sync.  It said manually import/ export.  ( see below)
I will have to look into how manual import and export work
"Can be used fully offline
Every action works fully offline. If you want your notes to be synced across all your devices then you can create absolutely free account in Clipto or stay unauthorized and use manual import/export capabilities. It is fully up to you."
https://play.google.com/store/apps/details?id=com.wb.clipboard.pro&hl=en_US

**Comment by atrashler** [comment]
> Apologies it did not say sync. It said manually import/ export. ( see below)
> I will have to look into how manual import and export work
> "Can be used fully offline
> Every action works fully offline. If you want your notes to be synced across all your devices then you can create absolutely free account in Clipto or stay unauthorized and use manual import/export capabilities. It is fully up to you."
> https://play.google.com/store/apps/details?id=com.wb.clipboard.pro&hl=en_US

Yes. It is about Android app. It supports import/export of your notes. And if you does not use authorisation then it does not sync your data (no internet activity at all).
Desktop app is the same, but does not support import/export yet.

**Comment by nomatica** [comment]
I see.  So it appears I would not be able to export  notes from android and then import them to desktop currently, but not immediately

**Comment by atrashler** [comment]
Yes. Now it is only possible to do by using cloud account.
I add features by request from the users. If import/export is required, I can add it. But it need some time to understand the reason why it is necessary (for example, it is less secured because it creates plain-text file that can be scanned by Google Drive on any other cloud) and to check for the right technical solution to implement and make it useful with or without an account.
